app/Interfaces/TenantInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Interfaces;
+
+class TenantInterface
+{
+    public const MODULE_CMS = 'CMS';
+    public const MODULE_CRM = 'CRM';
+    public const MODULE_EDU = 'EDU';
```

Comment:
Minor: Thinking we add REPORTS now as well? Is sth that will need adding soon anyway? Although it will be scope creep so nah